end-to-end-gee.Rmd

```diff
@@ -297,6 +297,20 @@ Earth Engine allows for exporting both vector and raster data to be used in an e
 ```
 [Open in Code Editor &#8599;](https://code.earthengine.google.co.in/?scriptPath=users%2Fujavalgandhi%2FEnd-to-End-GEE%3A01-Earth-Engine-Basics%2F08b_Export_(complete)){target="_blank"}
 
+Once you run this program, the *Tasks* tab will be highlighted. Click on it you will see the Exports will be read you be downloaded. Click *Run*. 
```

Comment:
"Exports will be read you be downloaded" - this doesn't make sense. Need to be corrected